src/openforms/authentication/contrib/digid_oidc/backends.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import SuspiciousOperation
+from django.urls import reverse
+
+from mozilla_django_oidc.auth import (
+    OIDCAuthenticationBackend as _OIDCAuthenticationBackend,
+)
+
+from openforms.authentication.constants import AuthAttribute
+
+from .mixins import SoloConfigMixin
+from .models import OpenIDConnectPublicConfig
+from .settings import OIDC_AUTHENTICATION_CALLBACK_URL
+
+logger = logging.getLogger(__name__)
+
+
+class OIDCAuthenticationDigiDBackend(SoloConfigMixin, _OIDCAuthenticationBackend):
+    """
+    Allows logging in via OIDC with DigiD
+    """
+
+    def __init__(self, *args, **kwargs):
+        self.config = OpenIDConnectPublicConfig.get_solo()
+
+        if not self.config.enabled:
+            return
+
+        super().__init__(*args, **kwargs)
+
+    def authenticate(self, *args, **kwargs):
+        # Differentiate between DigiD authentication via OIDC and admin login
+        # via OIDC by checking the callback url
+        if (
+            not self.config.enabled
+            or reverse(OIDC_AUTHENTICATION_CALLBACK_URL) != args[0].path
```

Comment:
what is `args[0]`?

Reply:
It's the `request`, not sure why I didn't simply call it `request`, I'll change that :sweat_smile: